File: Modul_5/Bahasa_C/PRAK504-2510817120005-NorKhalisyah.c
#include <stdio.h> 

int reverse(int n){ 
    int hasil = 0;
    while (n > 0) {
        int digit = n % 10;
        hasil = hasil * 10 + digit;
        n /= 10;
    }
    return hasil;
}

int main() { 
int A, B; 
printf("Masukkan nilai A dan B: ");
scanf("%d %d",&A,&B);

A=reverse(A);
B=reverse(B);

int C = A + B; 
printf("Hasil Representasi: %d",reverse(C)); 

return 0;
}

File: Modul_5/Bahasa_Python/PRAK504-2510817120005-NorKhalisyah.py
def reverse(n):
    hasil = 0
    while n > 0:
        digit = n % 10
        hasil = hasil * 10 + digit
        n //= 10
    return hasil

A, B = map(int, input("Masukkan nilai A dan B: ").split())

A = reverse(A)
B = reverse(B)

C = A + B
print("Hasil Representasi: ", reverse(C))